lib/vagrant-openshift/action/push_openshift_images.rb

```diff
@@ -70,11 +112,9 @@ def build_image(image_name, git_ref, repo_url, registry)
 git fetch && git checkout #{git_ref}
 git_ref=$(git rev-parse --short HEAD)
 echo "Building and testing #{image_name}-centos7:$git_ref ..."
-make test TARGET=centos7
-make build TARGET=centos7
+make test TARGET=centos7 VERSION=#{version}
```

Comment:
ah yes sorry i was thinking that was default behavior.  why isn't it default behavior?